files/ja/learn/javascript/first_steps/strings/index.md

```diff
@@ -69,7 +69,7 @@ l10n:
 1. JavaScript では、文字列を囲む引用符として、単一引用符と二重引用符を使用することができます。以下のどちらの例も正しく動きます。
 
    ```js
-   const sgl = '単一引用符';
+   const sgl = "単一引用符";
```

Comment:
This is an example comparing between a single quote and a double quote. So a single quote should be used in this case.